common/Jet_QA.C

```diff
@@ -344,22 +348,43 @@ R__LOAD_LIBRARY(libjetqa.so)
 	// --------------------------------------------------------------------------
 	inline std::vector<uint32_t> GetJetsToQA()
 	{
-
-		std::vector<uint32_t> vecJetsToQA = {
+	     std::vector<uint32_t> vecJetsToQA;
+	     if (HasCalos)
+	     {
+		   if (QA_SubtractedTowers)
+		   {	   
+                     vecJetsToQA.insert(vecJetsToQA.end(),  
+		     {
 			Type::AntiKtTowerSubR02,
 			Type::AntiKtTowerSubR03,
 			Type::AntiKtTowerSubR04,
-			Type::AntiKtTowerSubR05,
-			Type::AntiKtTowerR02,
+			Type::AntiKtTowerSubR05
+		     });
+		   }
+
+		   if (QA_RawTowers)
+                   {
```

Comment:
In line with above comment:
```suggestion
		   }
		   else
		   {
```